makefile

```diff
@@ -1,11 +1,17 @@
-.PHONY: build test docs env
+.PHONY: build test docs env all
+all:
+	make lint && make build && make test
 build:
 	python -m pip install -vv -e .
 test:
-	pytest --pyargs numba_rvsdg
+	coverage run -m pytest --pyargs numba_rvsdg
+	coverage report
 lint:
-	pre-commit run --all-files
+	pre-commit run --verbose --all-files
 docs:
 	cd docs && make html
 conda-env:
-	conda create -n numba-rvsdg python=3.12 python-graphviz pyyaml pytest sphinx sphinx_rtd_theme
+	conda create -n numba-rvsdg
+conda-install:
+	conda install python=3.12 python-graphviz pyyaml pytest sphinx sphinx_rtd_theme coverage pre-commit
+	pip install -U virtualenv
```

Comment:
@esc I wonder if `conda install virtualenv` will work too...

Reply:
I tried that, but when I use `pip` I get `virtualenv-20.26.1` anaconda.org only has `20.17.1` which seems to old.